### ISSUE TITLE
Change in Equilibrium Solutions

### ISSUE THREAD

**Issue by mtiller** [mtiller/ModelicaBook#463]
With the new tool versions, the plot titled "Equilibriums for rabbits, foxes and wolves" has changed.  The scale is very strange and it looks like we have a trivial solution for both the fox and rabbit populations.

**Comment by mtiller** [comment]
Equilibriums or Equilibria?

**Comment by dietmarw** [comment]
My vote is for Equilibria (just in order not to let Latin die out completely).

**Comment by gwr69** [comment]
Yes, please go for “equilibria”—I already crinch every time I read “the data is …” :)
